GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Branches.cs

```diff
@@ -112,10 +112,46 @@ protected void SelectRevision()
             }
         }
 
-        private sealed class LocalBranchNode : BaseBranchNode, IGitRefActions, ICanRename, ICanDelete
+        private class BaseBranchLeafNode : BaseBranchNode
```

Comment:
Should this be a separate file?
(For me this is fine, but there are other with more opinions reviewing later, you know who I mean.)

Reply:
LOL, are you fan of "_him who we do not name_" or "_he who must not be named_"?

Reply:
It would be good to break all nested classes into own files. I'm not going to block the PR for this though, it is a separate clean up task.

Reply:
OT: I know who reads and know all, so if I use the name or not makes no difference

Reply:
> It would be good to break all nested classes into own files.

I thought about it. But there were bugs to analyze as now again.
Perhaps I'll cleanup this in this PR later.